src/components/ContextData/index.jsx

```diff
@@ -0,0 +1,18 @@
+import React,{ createContext, useContext, useState } from 'react'
+import data from '../../data'
+
+export const ContextData = createContext()
+
+export const DataProvider = ({ initalState = {}, children }) => {
```

Comment:
Customer wrapper for the Context Provider. This allows us to add our `values` state and setter to the context.
You can set an initial state if you like but it will default to an empty object.